File: JANUARY/25-01-2026/1984.kt
class Solution {
    fun minimumDifference(nums: IntArray, k: Int): Int {
        if (k <= 1) return 0
        nums.sort()
        var ans = Int.MAX_VALUE
        for (i in 0..nums.size - k) {
            ans = minOf(ans, nums[i + k - 1] - nums[i])
        }
        return ans
    }
}

File: JANUARY/25-01-2026/1984.go
func minimumDifference(nums []int, k int) int {
    if k <= 1 {
        return 0
    }
    sort.Ints(nums)
    ans := math.MaxInt32
    for i := 0; i+k-1 < len(nums); i++ {
        if nums[i+k-1]-nums[i] < ans {
            ans = nums[i+k-1] - nums[i]
        }
    }
    return ans
}